submit-api/src/submit_api/services/package.py

```diff
@@ -272,8 +272,12 @@ def update_submission_status(package, status, session):
             raise BadRequestError("Invalid status")
         submissions = [submission for item in package.items for submission in item.submissions]
         for submission in submissions:
-            submission.status = status
-            session.add(submission)
+            if submission.status == SubmissionStatus.PENDING_REPLACEMENT:
+                submission.active = False
+                submission.status = status
```

Comment:
this should probably be "Replaced" no?

Reply:
we can add that when its set to active false its gone entirely except for previous versions but yes we can add that in